pandas/io/excel/_base.py

```diff
@@ -647,7 +647,7 @@ class ExcelWriter(metaclass=abc.ABCMeta):
     """
     Class for writing DataFrame objects into excel sheets.
 
-    Default is to use xlwt for xls, openpyxl for xlsx, odf for ods.
+    Default is to use xlwt for xls, xlsxwriter for xlsx, odf for ods.
```

Comment:
For `xlsx`, appears that `xlsxwriter` is preferred over `openpyxl` if installed. Might want to clarify that specifically here.

```
    _default_writers = {
        "xlsx": "openpyxl",
        "xlsm": "openpyxl",
        "xlsb": "pyxlsb",
        "xls": "xlwt",
        "ods": "odf",
    }
    assert mode in ["reader", "writer"]
    if mode == "writer":
        # Prefer xlsxwriter over openpyxl if installed
        xlsxwriter = import_optional_dependency("xlsxwriter", errors="warn")
        if xlsxwriter:
            _default_writers["xlsx"] = "xlsxwriter"
        return _default_writers[ext]
```

Reply:
Do you want me to change the pandas.ExcelWriter doctring this way :

    Class for writing DataFrame objects into excel sheets.

    Default is to use :
    * xlwt for xls
    * xlsxwriter for xlsx (xlsxwriter is preferred over openpyxl if both installed)
    * odf for ods.
    See DataFrame.to_excel for typical usage.

Reply:
Sure.

Maybe clearer is `xlsxwriter for xlsx if xlsxwriter is installed otherwise openpyxl`